Makefile

```diff
@@ -15,7 +15,7 @@ policy.wasm: $(SOURCE_FILES) go.mod go.sum types_easyjson.go
 		--rm \
 		-e GOFLAGS="-buildvcs=false" \
 		-v ${PWD}:/src \
-		-w /src tinygo/tinygo:0.27.0 \
+		-w /src tinygo/tinygo:0.28.1 \
```

Comment:
(opening thread here randomly)

I think we can drop the `types_easyjson.go` target (and remove it as deps from the `test` target)